### PR TITLE
Bumped version to 3.0.0

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

**Comment by claudep** [comment]
@hugovk What about releasing today? Any other short-term plan?

**Comment by hugovk** [comment]
Yes, I think we're ready for a release!

We've dropped Python 3.5. Are we using semver and does this qualify for a major bump?

**Comment by claudep** [comment]
I don't care so much, but if you think we should rather release 3.0.0, I'm fine with that.

File: HISTORY.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # History
 
-## Unreleased
+## 2.1.0 (2020-12-05)
```
Comment (hugovk) [inline HISTORY.md:3]: Okay, let's bump to 3.0.0, might save some surprises for people.

```suggestion
## 3.0.0 (2020-12-05)
```

Would you like me to do the release or would you like to?

Comment (claudep) [reply HISTORY.md]: If you have time for it, go!

Comment (hugovk) [reply HISTORY.md]: 🚀

**Pull request merged**